src/content/hinting.ts

```diff
@@ -630,7 +637,11 @@ class Hint {
         this.flag.style.cssText = `
             top: ${window.scrollY + top}px !important;
             left: ${window.scrollX + left}px !important;
+            ${extrastyle}
```

Comment:
When I thought about implementing this I was thinking more about adding extra "semantic" classes to the hints that themes could choose to display however they want. Are there issues with this approach?